src/app/shared/components/form-field/form-field.tsx

```diff
@@ -5,9 +5,17 @@ import * as ReactForm from 'react-form';
 
 require('./form-field.scss');
 
+export function getNestedField(src: any, path: string) {
```

Comment:
Wondering if we could use a return value type.  Otherwise, looks good!

Reply:
done. Explicitly specified return type as any in getNestedField function